tests/ctst/steps/azureArchive.ts

```diff
@@ -407,11 +443,80 @@ Then('object {string} should expire in {int} days', async function (this: Zenko,
     assert(diff >= realTimeDays && diff < realTimeDays + 0.005);
 });
 
+Given('an azure archive location {string}', async function (this: Zenko, locationName: string) {
+    const locationConfig = {
+        name: locationName,
+        locationType: 'location-azure-archive-v1',
+        details: {
+            endpoint: AZURE_STORAGE_BLOB_URL,
+            bucketName: this.parameters.AzureArchiveContainer,
+            queue: {
+                type: 'location-azure-storage-queue-v1',
+                queueName: this.parameters.AzureArchiveQueue,
+                endpoint: AZURE_STORAGE_QUEUE_URL,
+            },
+            auth: {
+                type: 'location-azure-shared-key',
+                accountName: this.parameters.AzureAccountName,
+                accountKey: this.parameters.AzureAccountKey,
+            },
+        },
+    };
+    const result = await this.managementAPIRequest('POST', `/config/${this.parameters.InstanceID}/location`, {},
+        locationConfig);
+    assert.strictEqual(result.statusCode, 201);
+    this.addToSaved('locationName', locationName);
+    await Utils.sleep(60000); // Wait for location to be updated
```

Comment:
Don't we have an async way to test it, so that we don't wait for nothing, or not wait enough in case of slow CI?
Otherwise, maybe worth defining this duration as a const as it seems to be used multiple times

Reply:
This could be possible if we had a simple access to kube to check the status of the pods, unfortunately this is not the case yet so we "just" wait..

Reply:
And this is not the purpose of this ticket to add a way to check pod's status

Reply:
Here is the ticket, if you want to add a `// TODO` : https://scality.atlassian.net/browse/CTST-35